modules/launcher/src/main/java/org/mule/runtime/module/launcher/MuleContainer.java

```diff
@@ -71,6 +76,21 @@
 
 public class MuleContainer {
```

Comment:
How can this be tested? Can we create some system tests for this? 
Even if they are not run everytime? I don't know if this can be tested with munit? Probably a cluster hast to be created.